packages/expo-location/android/src/main/java/expo/modules/location/services/LocationTaskService.java

```diff
@@ -20,8 +20,8 @@
 public class LocationTaskService extends Service {
   private static final String TAG = "LocationTaskService";
   private static int sServiceId = 481756;
-
   private String mChannelId;
+  private String killService = "false";
```

Comment:
should it be `boolean` because you are expecting boolean 🤔 ? and change `getString` to `getBoolean` 🤔 ?

Reply:
Yes, you're right. I will apply your changes ;) Thanks